app/services/catalog/share_resource.rb

```diff
@@ -0,0 +1,23 @@
+module Catalog
```

Comment:
@gmcculloug This looks like a new file but its the same as 
https://github.com/RedHatInsights/catalog-api/blob/master/app/services/api/v1x0/catalog/share_resource.rb
Should this service object move into v1x1 or update the one in v1x0